packages/code-studio/src/AppRoot.tsx

```diff
@@ -6,6 +6,11 @@ import { DownloadServiceWorkerUtils } from '@deephaven/iris-grid';
 import MonacoWorker from 'monaco-editor/esm/vs/editor/editor.worker?worker';
 import AppRouter from './main/AppRouter';
 
+// load addional css for playwright docker tests
+if (import.meta.env.VITE_PLAYWRIGHT === '1') {
+  await import('./Playwright.css');
+}
```

Comment:
Doesn't matter since the file is so small, but this always gets included in the prod bundle. Just not loaded without this flag at buildtime. It should also get split into its own chunk that is never loaded.

Reply:
Yeah, I didn't know how do exclude from from a prod build, it doesn't get loaded but would still exist on the server. But it's also tiny, so 🤷

Reply:
I'm pretty sure this actually gets removed from the prod bundle. The env check should get statically replaced in the build, and tree shaking should remove it. Based on some testing, I think it still exists in the source map but gets removed from the .js code.